src/index.html

```diff
@@ -6,9 +6,44 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="#">
+          <img src="images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav class="menu">
+        <ul class="menu-list">
+          <li class="menu-item">
+            <a href="#" class="is-active">Apple</a>
+          </li>
+          <li class="menu-item">
+            <a href="#">Samsung</a>
+          </li>
```

Comment:
```suggestion
        <ul class="menu-list">
          <li class="menu-item">
            <a href="#" class="is-active">Apple</a>
          </li>
          
          <li class="menu-item">
            <a href="#">Samsung</a>
          </li>
```
add blank lines between blocks of code